packages/block-library/src/quote/edit.js

```diff
@@ -82,6 +82,8 @@ export default function QuoteEdit( {
 				{ ( ! RichText.isEmpty( citation ) || isSelected ) && (
 					<RichText
 						identifier="citation"
+						tagName="cite"
+						display={ { style: 'inline-block' } }
```

Comment:
doesn't look like this is working on my end:

<img width="448" alt="Screenshot 2021-03-24 at 14 40 47" src="https://user-images.githubusercontent.com/3593343/112320180-11c1a180-8caf-11eb-9ee1-78fba2468372.png">

Reply:
oops inverted style and display haha :)

Reply:
Now I see a new difference with the frontend, since the citation doesn't have the display property defined on the frontend:

Editor | Frontend
--- | ---
<img width="402" alt="Screenshot 2021-03-24 at 15 10 27" src="https://user-images.githubusercontent.com/3593343/112324620-4899b680-8cb3-11eb-8626-13181d3f4b69.png"> | <img width="337" alt="Screenshot 2021-03-24 at 15 10 37" src="https://user-images.githubusercontent.com/3593343/112324614-48012000-8cb3-11eb-8eda-46cf71f1fb02.png">

Reply:
there were a margin top that didn't apply on the frontend because the cite was inline, so I just removed that style making the difference between inline and inline-block not visible.

Reply:
Now they both look the same to me, thank you!